pkgs/tools/networking/davix/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, openssl, libxml2, boost }:
+
+stdenv.mkDerivation rec {
+  name = "davix-0.4.0";
+  buildInputs = [ stdenv pkgconfig cmake openssl libxml2 boost ];
+
+  src = fetchFromGitHub {
+	owner = "cern-it-sdc-id";
+	repo = "davix";
+    rev = "9d8f400ec1882602fc18312f35d617fe94ebbd67";
```

Comment:
It is better to use the version tag instead of the commit as it is visible that we ware actually fetching the version advertised in the name.